src/utils/logger.js

```diff
@@ -10,20 +10,38 @@ const LOG_LEVEL: { [level: string]: Object } = {
   "OFF": JsLogger.OFF
 };
 
-class LoggerFactory {
-  constructor(options?: Object) {
-    JsLogger.useDefaults(options || {});
-  }
+JsLogger.useDefaults({defaultLevel: JsLogger.ERROR});
 
-  getLogger(name?: string) {
-    if (!name) {
-      return JsLogger;
-    }
-    return JsLogger.get(name);
+/**
+ * get a logger
+ * @param {?string} name - the logger name
+ * @returns {Object} - the logger class
+ */
+function getLogger(name?: string): Object {
+  if (!name) {
+    return JsLogger;
   }
+  return JsLogger.get(name);
+}
+
+/**
+ * get the log level
+ * @param {?string} name - the logger name
+ * @returns {Object} - the log level
+ */
+function getLogLevel(name?: string): Object{
+  return getLogger(name).getLevel();
 }
 
-const lf = new LoggerFactory({defaultLevel: JsLogger.DEBUG});
+/**
+ * sets the logger level
+ * @param {Object} level - the log level
+ * @param {?string} name - the logger name
+ * @returns {void}
+ */
+function setLogLevel(level: Object, name?: string): void{
+  getLogger(name).setLevel(level);
+}
 
-export default lf;
-export {LOG_LEVEL};
+export default getLogger;
+export {LOG_LEVEL, getLogLevel, setLogLevel};
```

Comment:
change the var name `LOG_LEVEL` to `LogLevel` since it's an enum and then we can export it regularly without use the `as` syntax of flow in other files

Reply:
@dan-ziv it's either an enum or not.
If it's an enum we should keep the convention as we do with other enums, like:
`import {PLAYER_EVENTS as PlayerEvents, HTML5_EVENTS as Html5Events, CUSTOM_EVENTS as CustomEvents} from './event/events'
`

Reply:
Those conventions are wrong and already fixed in my player typing PR which you all ignore of reviewing :)
From what I see in all the lead projects the correct conventions for enums should be lead with capitals (i.e. MyEnum) and constants should be all written in capital letters (i.e. MY_CONST)

Reply:
fixed